example/go.mod

```diff
@@ -0,0 +1,56 @@
+module github.com/kitex-contrib/config-file/example
```

Comment:
The new repository https://github.com/kitex-contrib/examples is a better place for such examples.   Please submit another pull request with these example files under `config/file` to this repo.